Bottom: f510b7d3a1caaff319b2471b1481c9e2186e1646
Top:    3acfdcc701d1b89b4c9005f3a661b98ae380f441
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-11 13:44:28 +0800

Refresh of autofs-5.0.7-fix-fix-map-entry-duplicate-offset-detection.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index e15aa1f..39d7889 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -47,6 +47,7 @@
 - dont probe rdma mounts.
 - fix master map mount options matching.
 - fix master map bogus keywork match.
+- fix fix map entry duplicate offset detection.
 
 25/07/2012 autofs-5.0.7
 =======================
